examples/instanced_shapes/src/lib.rs

```diff
@@ -0,0 +1,19 @@
+#[allow(dead_code)] // Prevent warning about the free functions in main.
```

Comment:
I added this here to prevent the build from complaining about the helper functions in the `main.rs` file. Not too happy about it, but didn't see another way.

Reply:
I simplified the transformation calculations quite a lot so it's not necessary anymore, but if it was, I think it's perfectly all right, it's just an example 🙂

Reply:
Yes, this is much much better :ok_hand: . Good call to specify the number of cubes on the sides instead of the total :+1:

Reply:
Thanks 👍 It made everything a bit easier 🙂